controllers/front/notify.php

```diff
@@ -181,23 +195,19 @@ public function postProcess()
                 die('canceled');
             }
 
-            PrestaShopLogger::addLog(
-                sprintf(
-                    '%s has caught an error: %s',
-                    __CLASS__,
-                    $e->getMessage()
-                ),
-                1,
-                null,
-                null,
-                null,
-                true
-            );
+            /** @var LoggerInterface $logger */
```

Comment:
Put it just below catch to make sure it logs error as I see couple die methods here and it might be unreachable